### DISCUSSION TITLE
guide/components/link-card/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#159] [docs-comment]
# guide/components/link-card/

概述 使用 <LinkCard> 组件在页面中显示链接卡片。 Props 插槽 示例 输入： 输出：

https://theme-plume.vuejs.press/guide/components/link-card/

**Comment by EternityXuNuo** [comment]
不能实现卡片内段落内换行吗

**Reply by pengzhanbo** [reply]
可以的，直接将 描述作为 组件的 子元素 即可，可以随意嵌入任意内容。但目前发现 子元素 存在间距过大的问题，将会在下版本中进行调整优化。

**Comment by dingyuqi** [comment]
我发现链接卡片除了尾部的右箭头还会存在一个外部链接的箭头在标题旁边.我不知道是否能够去除这个, 看到你的demo里面似乎没有这个额外的斜向上的箭头
![image](https://github.com/user-attachments/assets/7593b64f-d9f8-4d07-a021-4e382840d42a)


代码如下:
```
<CardGrid>
  <LinkCard title="Node" icon="logos:nodejs-icon" href="https://nodejs.org/" />
  <LinkCard title="Deno" icon="logos:deno" href="https://deno.land/" />
  <LinkCard title="Bun" icon="logos:bun" href="https://bun.sh/" />
</CardGrid>
```

**Reply by pengzhanbo** [reply]
该问题应该是在 `rc.99` 版本中有做修复，但我遗漏了 `LinkCard` 组件，我会在下个版本修复它。

**Comment by dingyuqi** [comment]
请问 `herf` 是否能够跳转到相关的资源上面. 比如 `public` 文件夹下的某个 pdf 文件. 我发现我在 `herf` 中使用相对路径, 点击跳转之后页面会自动在路径后自动添加 `.html` 后缀导致了 404 错误. 我知道这也许不属于一个 Bug, 只是我使用的问题. 我想也许我需要将 `herf` 写为带域名的绝对路径, 我知道这样是可以的.

#### 使用场景
存在 `public/pdf/certificate.pdf`, 然后代码如下:

```
<LinkCard title="Systems Analyst" href="/pdf/certificate.pdf"  icon="/icon/ruankao.png">
</LinkCard>
```

**Reply by pengzhanbo** [reply]
对于此类链接，最好是加上 target="_blank"，打开新的标签页。
